lib/cli/src/cli.rs

```diff
@@ -344,14 +344,34 @@ fn test_split_version() {
     );
 }
 
+#[derive(Debug)]
+pub(crate) enum SplitVersionError {
+    InvalidCommandName(anyhow::Error, Option<anyhow::Error>),
+    Other(anyhow::Error),
+}
+
+impl std::error::Error for SplitVersionError {}
```

Comment:
Can you implement the `source()` method here? That way the underlying error will be added to the error chain when pretty-printing errors.

Reply:
I refactored the error handling, so this does no longer apply.